packages/thumbprint-react/components/UIAction/get-button-props.ts

```diff
@@ -1,3 +1,13 @@
+interface ButtonProps {
+    type: 'button' | 'submit';
+    onClick?: () => void;
```

Comment:
Are these not supposed to include the event as a parameter? He's `onClick`, for example:

```
(event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
```

Reply:
yes, i wrote this code before i knew the react dom types so well. fixed here and elsewhere.